ssc/core.cpp

```diff
@@ -226,7 +217,7 @@ bool compute_module::evaluate()
             }
             default:
             {
-                float time = -1.;
+//                float time = -1.;
                 //log(variable_name + " of data type " + var_data::type_name(variable_data->type) + " is not supported for ssc_equations", SSC_ERROR, time);
```

Comment:
Why is this commented out? Ben Elliston mentioned this and was noted on his pull request

Reply:
I think throwing an error is a good idea! I'll add that in. I commented out the line since it doesn't do anything.

Reply:
Uncommented out the error message. Will merge once Travis tests pass